### DISCUSSION TITLE
Where's the chat icon?

### DISCUSSION THREAD

**Discussion by Neil-Network** [community/community#53716] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Using updated insider edition. I have an active subscription for copilot. Why is there no chat icon as shown in all the tutorials? 
![image](https://user-images.githubusercontent.com/84641315/234380073-7fa30e0f-b8e1-429e-94dd-2c2b7eb97279.png)

**Comment by NJacksonDev** [comment]
If this is the same issue that others were having in March, the chat feature could just not be enabled yet for all users. See link to discussion below.

https://github.com/orgs/community/discussions/50851

**Comment by davecheney** [comment, marked as answer]
@Neil-L-1 it don't see your name on the Chat waitlist. You will need to sign up to the waitlist, then wait to be admitted before you can use the chat feature.

**Reply by Neil-Network** [reply]
Thanks. I was assuming by paying for the subscription there wasn’t a wait list anymore.

**Reply by davecheney** [reply]
https://github.com/features/preview/copilot-x
